core/validation/specific_validations/towns_fund_round_four.py

```diff
@@ -138,7 +141,7 @@ def validate_funding_profiles_funding_source(
         return [
             TownsFundRoundFourValidationFailure(
                 sheet="Funding",
-                section=f"Project Funding Profiles - Project {get_project_number(row['Project ID'])}",
+                section=f"Project Funding Profiles - Project {get_project_number_by_cell(row.name, 'Funding')}",
```

Comment:
so just to confirm - this `row.name` doesn't need `+ 2`?

Reply:
No, because Joel added a +2 on all the indexes before validation